docker-compose.dev.yml

```diff
@@ -14,6 +14,10 @@ services:
       args:
         USER_UID: ${USER_UID}
         SDK_VERSION: ${SDK_VERSION:-latest}
+        NEWRELIC_ENABLED: ${NEWRELIC_ENABLED}
```

Comment:
Maybe make false by default for more easy dev installation

Reply:
it is an obolete code. removed